app/views/users/_profile_tabs.html.erb

```diff
@@ -0,0 +1,13 @@
+      <ul class="nav nav-tabs" id="myTab">
+        <li class="active"><a href="#contributions"><i class="glyphicon glyphicon-star"></i> Contributions</a></li>
+        <li><a href="#profile"><i class="glyphicon glyphicon-info-sign"></i> Profile</a></li>
+        <% if current_user && (current_user.admin? || @user == current_user) %>
```

Comment:
Thinking out loud, but could we approach this with cancan instead?

something like ```if can? :view_activity```, and then add that logic to the ability file?